fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -1339,10 +1337,13 @@ private void computeFragmentHosts() throws Exception {
         }
     }
 
+    static final int BUCKET_ABSENT = 2147483647;
```

Comment:
why choose this number?

Reply:
Is it possible that the hash value of other buckets may conflict with this one?

Reply:
never, 
hash_value=hash_value % num_buckets;
hash_value=bucketseq_to_partition[hash_value];
so at last the hash_value is partition index,  BUCKET_ABSENT, means that the bucket is pruned, have no partition index to map into.